samples/ImplicitFlow/AuthorizationServer/Startup.cs

```diff
@@ -0,0 +1,142 @@
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILogger<Startup> logger)
+        {
+            app.UseIdentity();
+            app.UseOAuthValidation();
+
+            app.UseCors(builder => builder
+                .AllowAnyHeader()
+                .AllowAnyOrigin()
+                .AllowAnyMethod());
+
+            app.UseOpenIddict();
+            app.UseMvcWithDefaultRoute();
+
+            // In a production app, seed this in a setup tool.
+            SeedDatabase(app, env);
+        }
+
+        private void SeedDatabase(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            var options = app
+                .ApplicationServices
+                .GetRequiredService<DbContextOptions<ApplicationDbContext>>();
+
+            using (var context = new ApplicationDbContext(options))
+            {
+                // drop and recreate
+                context.Database.EnsureDeleted();
+                context.Database.EnsureCreated();
+
+                if (!context.Applications.Any())
+                {
+                    var aureliaOidcHost = "http://localhost:9000";
+
+                    context.Applications.Add(new OpenIddictApplication<Guid>
+                    {
+                        ClientId = "Aurelia.OidcClientJs",
+                        DisplayName = "Aurelia Oidc Client Js",
+                        LogoutRedirectUri = $"{aureliaOidcHost}/signout-oidc",
+                        RedirectUri = $"{aureliaOidcHost}/signin-oidc",
+                        Type = OpenIddictConstants.ClientTypes.Public
+                    });
+
+                    var oidcClientHost = "http://localhost:5000";
+
+                    context.Applications.Add(new OpenIddictApplication<Guid>
+                    {
+                        ClientId = "OidcClientJs.OidcClient",
+                        DisplayName = "Oidc Client Js - Oidc Client Sample",
+                        LogoutRedirectUri = $"{oidcClientHost}/oidc-client-sample.html",
+                        RedirectUri = $"{oidcClientHost}/oidc-client-sample.html",
+                        Type = OpenIddictConstants.ClientTypes.Public
+                    });
+
+                    context.Applications.Add(new OpenIddictApplication<Guid>
+                    {
+                        ClientId = "OidcClientJs.UserManager",
+                        DisplayName = "Oidc Client Js - User Manager Sample",
+                        LogoutRedirectUri = $"{oidcClientHost}/user-manager-sample.html",
+                        RedirectUri = $"{oidcClientHost}/user-manager-sample.html",
+                        Type = OpenIddictConstants.ClientTypes.Public
+                    });
+
+                    context.Applications.Add(new OpenIddictApplication<Guid>
+                    {
+                        ClientId = "ResourceServer01",
+                        Type = OpenIddictConstants.ClientTypes.Confidential
```

Comment:
I'm really surprised your sample works. These apps are marked as confidential but don't have any client secret... this cannot work as OpenIddict requires client authentication for introspection.